Runtime/Scripts/Schema/Sampler.cs

```diff
@@ -121,10 +121,22 @@ public enum WrapMode
             }
         }
 
-        public void Apply(Texture2D image) {
+        public void Apply(Texture2D image, 
+                          int defaultMinFilter=(int)MinFilterMode.NearestMipmapLinear, 
+                          int defaultMagFilter=(int)MagFilterMode.Linear) {
             if (image == null) return;
             image.wrapModeU = wrapU;
             image.wrapModeV = wrapV;
+
+            // Use the default filtering mode for textures that have no such specification in data
+            if (minFilter == (int)MagFilterMode.None)
```

Comment:
This method applies this Sampler's settings onto `image`, but now also potentially alters it (if some filter was `None` before). I think there should be a clear separation.

Can you please extract the minFilter/maxFilter to filterMode conversion that the `filterMode` getter is doing into a static method and re-use it here? You can then fall back to `defaultMagFilter` without changing `magFilter`.